docs/pages/technical-specs/expo-updates-0.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Expo Updates
+sidebar_title: Expo Updates
+---
+
+version 0
+
+---
+
+## Introduction
+
+This is the specification for Expo Updates, a protocol for delivering updates to apps running on multiple platforms.
+
+### Conformance
+
+Conforming servers and client-libraries must fulfill all normative requirements. Conformance requirements are described in this document by both descriptive assertions and key words with clearly defined meanings.
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in the normative portions of this document are to be interpreted as described in [IETF RFC 2119](https://tools.ietf.org/html/rfc2119). These key words may appear in lowercase and still retain their meaning unless explicitly declared as non‐normative.
+
+A conforming implementation of this protocol may provide additional functionality, but must not where explicitly disallowed or would otherwise result in non‐conformance.
+
+### Overview
+
+An _update_ is defined as a [_manifest_](#manifest-response) together with the assets referenced inside the manifest.
+Expo Updates is a protocol for delivering updates to apps running on multiple platforms.
+
+An app running a conformant Expo Updates client-library will fetch the most recent update from a conformant update server. If the client-library cannot fetch an update or if it already has the most recent update, the client-library will load the update saved in the client-library's cache.
+
+The following describes how a client-library will interact with a conforming updates server:
+1. The client-library will make a [request](#request) for the most recent manifest. 
+2. If it is a new manifest, the client-library will proceed to make requests to download and store the assets specified in the manifest.
+3. Finally, the client-library will update its local state according to any metadata provided by the response [headers](#headers).
+
+We anticipate the primary user of this spec will be organizations who need to manage their own update server to satisfy internal requirements.
+
+## Manifest Request
+A conformant client-library MUST make a GET request with the following headers:
+```
+accept: application/json
+expo-platform: string
+expo-runtime-version: string
+```
+along with any headers stipulated by a previous responses' [server defined headers](#headers):
+
+* `expo-platform` MUST be platform type the client is running on: 
+    * iOS MUST be `expo-platform: ios`
+    * Android MUST be `expo-platform: android`
+* `expo-runtime-version` MUST be the runtime version the client is running on.
```

Comment:
It may be worth expanding upon what a runtime version is and how it is expected to be specified by a client implementation.